app/views/proposals/detail/_observer.html.haml

```diff
@@ -0,0 +1,22 @@
+.card-for-observers
+  .row
+    .medium-12.column
+      .card
+        .card-head.column
+          %h2
+            Observers
+        .content-content.column
+          %ul.observer-list
+            - @subscriber_list.each do |user, role_str, observation|
+              = render partial: "proposals/details/observer_row", locals: { user: user, role_str: role_str, observation: observation, proposal: proposal }
+          - if policy(Observation.new(proposal: proposal)).can_create?
```

Comment:
let's also create an `observation` that is assigned to `@proposal.observations.new` in the controller. then we can use that here and on L 13!